deploy/crontab.yml

```diff
@@ -0,0 +1,81 @@
+---
+- hosts: ynr_cron
```

Comment:
This line needs to be `prod_cron`, and it requires us to set the right tag on the EC2 instance before we deploy this:

See how `hosts` are defined here: https://github.com/DemocracyClub/yournextrepresentative/blob/master/deploy/servers.aws_ec2.yml

You can see this playing out by doing the following:

Make sure your AWS profile can connect to the "Sym Roe" account, then

```cd deploy
ansible-inventory -i servers.aws_ec2.yml --graph
```

Reply:
Now fixed.